SoftLayer/managers/network.py

```diff
@@ -126,9 +126,11 @@ def add_subnet(self, subnet_type, quantity=None, vlan_id=None, version=4,
         if version == 4:
             if subnet_type == 'global':
```

Comment:
global ips are not a thing anymore, we can likely remove these bits to make this function a little simpler.